src/tools/Makefile.am

```diff
@@ -27,6 +27,8 @@ cgcreate_CFLAGS = $(CODE_COVERAGE_CFLAGS)
 libcgset_la_SOURCES = cgset.c tools-common.c tools-common.h
 libcgset_la_LIBADD = $(CODE_COVERAGE_LIBS)
 libcgset_la_CFLAGS = $(CODE_COVERAGE_CFLAGS) -DSTATIC= -DUNIT_TEST
+libcgset_la_LDFLAGS = -Wl,--no-undefined $(top_builddir)/src/libcgroup.la \
+		      -lpthread
```

Comment:
Looks good to me. Just a style preference comment, can it be simplified more by using the LADD? using libcgset_la_LDFLAGS = -Wl,--no-undefined $(LDADD)

Reply:
Good idea.  Thanks, Kamalesh.